dateparser/custom_lang_detect/__init__.py

```diff
@@ -0,0 +1,21 @@
+import importlib
+from dateparser.conf import apply_settings
+
+
+class CustomLanguageDetectCache:
+    loaded_parser = None
+    detect_languages_module = None
+
```

Comment:
Is caching as this do able ?

Reply:
I think I answered below https://github.com/scrapinghub/dateparser/pull/932/files#r648054376